models/balancer/arbitrum/balancer_v2_arbitrum_liquidity.sql

```diff
@@ -20,10 +20,11 @@ WITH pool_labels AS (
         SELECT
             date_trunc('day', minute) AS day,
             contract_address AS token,
+            decimals,
             AVG(price) AS price
         FROM {{ source('prices', 'usd') }}
         WHERE blockchain = "arbitrum"
-        GROUP BY 1, 2
+        GROUP BY 1, 2, 3
```

Comment:
We may need to rewrite the group by in full column names.

Reply:
Substring function may also varies.

Reply:
should I do this change? how do I make it compatible with both Spark and Dune SQL if I use built-in functions for e.g. generating a time series that is different in each engine?

Reply:
are PRs only merged if the spell is compatible with both engines from now onwards? that's not clear to me too